rules/linux/privilege_escalation_setgid_bit_set_via_chmod.toml

```diff
@@ -52,7 +52,7 @@ reference = "https://attack.mitre.org/techniques/T1166/"
 
 
 [rule.threat.tactic]
-id = "TA0004"
+id = "TA0003"
```

Comment:
there were quite a few tactic IDs improperly mapped. New unit test checks for this

Reply:
the links aren't fixed though

Reply:
nice catch - will update and add checks in the unit tests

Reply:
ada9043d3dc67a67bc67a308329b7bc3200e8746